src/customers/account-dashboard-reorders.md

```diff
@@ -22,6 +22,9 @@ Customer is redirected to the [shopping cart]({% link quick-tour/shopping-cart.m
 
 **Case 2.** Some/all products from the order are not available for reorder
 
+{:.bs-callout-info}
+It is possible to reorder `Not Visible Individually` products
```

Comment:
Please add a period at the end of the sentence for each instance.

Reply:
Done